runners/spark/src/main/java/org/apache/beam/runners/spark/SparkRunner.java

```diff
@@ -453,7 +454,9 @@ TransformEvaluator<TransformT> translate(
       } else {
         pValues = node.getInputs();
       }
-      PCollection.IsBounded isNodeBounded = isBoundedCollection(pValues.values());
+      // FIXME Why is the output of TestSparkRunner.UnboundedReadFromBoundedSourceOverrideFactory BOUNDED!?!
```

Comment:
@aromanenko-dev @echauchot A bit stuck here.... Wondering if you have any idea what the issue is, see the replacement below. It replaces a bounded read with an unbounded read so validates runner tests using `Create.of(...)` can be run in streaming mode.

However, `isBounded` of the output still returns `BOUNDED` 🤯 

https://github.com/apache/beam/pull/22473/files#diff-d81f49eb0330230bd03ce6cd33b5f70f59c443aac57741e877ececbada32b16bR112-R117